jest.config.js

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
Jest auto-generated this file.

Reply:
Sweeeet, thanks a lot, that's awesome!!
I'll have a look and get it merged